company/okrs/index.md

```diff
@@ -0,0 +1,92 @@
+
+Each OKR has the following format:
+
+`Title or team: Objective as 1 sentence. Key result, key result, key result. => Outcome, outcome, outcome.`
+
+1. The `Title or team` is the title (e.g., `CEO` or `VP Engineering`) of the person who is directly responsible or the name of the team responsible.
+   - The first 2 levels of OKRs all have a directly responsible individual. Deeper levels can have teams. (For functions where we don't yet have a management structure in place, we may violate this.)
+1. The `Objective as 1 sentence.` is a pithy, informal summary of the objective, not an exhaustive and precise definition or description of the implementation. For example:
+   - Good: `Automate time-consuming ops and support tasks.`
+   - Bad: `Write scripts for headless browser testing of release grid items and update sourcegraph/sourcegraph and sourcegraph/deploy-sourcegraph CI pipelines to execute these scripts.`
+1. The `Key result, key result, key result.` is a list of 1-3 key results (the fewer the better) that have precise definitions.
+   - Quantitative key results are ideal, but if they aren't appropriate for the objective, don't try to force it. Instead, choose a key result that is clear in its reliance on someone's judgment.
+   - If a key result has a complex definition (more than a few words), link to a handbook page with the precise definition instead of inlining it in the OKR list.
+   - Key results can be links to issues or [RFCs](../../handbook/engineering/rfcs/index.md).
+   - Examples:
+      - Good (quantitative): `$M net new ARR`, `100% of monthly releases ship on time`
+	  - Good (human judgment): `Support LSIF-based code intel in targeted languages (Go, JavaScript/TypeScript, Python, Java, C#, and C/C++)` because quantitatively defining this has proven to be unproductive
+	  - Bad: `Create an effective pitch for Fortune 500 prospects` because "effective" is not defined and different people are likely to have a wide variety of opinions about what it means
+1. The `=> Outcome, outcome, outcome.` part is only added/updated after the quarter starts.
+1. Top-level CEO OKRs may just be short phrases (such as `Net new ARR`) instead of listing out key results when those key results would be redundant with the sub-OKRs.
+
+### Sensitive information
+
+Do not include any specific financial numbers, customer names, or any other sensitive information in the public OKR document. Instead, use placeholders (such as `N` and `M` for 2 numbers or `C1` and `C2` for 2 customers) and document their meanings in the linked `OKRs YYYY-QN` Google Doc's `Key for sensitive information` section.
+
+### Changing OKRs mid-quarter
+
+If your OKR no longer accurately communicates what's important, why, and how it'll get done, then you should change it even if the quarter has already started. We want to minimize mid-quarter OKR changes as much as possible, but sometimes it's the least bad course of action.
```

Comment:
My inclination would be to ~~strikethrough~~ Objectives or KRs that are no longer relevant (rather than deleting them). In otherwords, prefer delete + add instead of edit.

Reply:
This makes the doc look messy, which makes it hard to read and I fear will cause people to not update it as diligently. We can always find the history in git (easier IMO than finding it in gdocs).